lms/djangoapps/verify_student/tests/test_views.py

```diff
@@ -1390,6 +1484,36 @@ def _assert_user_name(self, full_name):
         account_settings = get_account_settings(self.user)
         self.assertEqual(account_settings['name'], full_name)
 
+    @contextlib.contextmanager
+    def _intercept_post(self):
+        """
+        Mock `requests.post()` to always return a 200 status code.
+
+        Yields:
+            mock.Mock
+
+        """
+        with patch("verify_student.models.requests.post") as mock_post:
+            mock_post.return_value = mock.Mock(
+                status_code=200,
+                text="Fake response",
+            )
+            yield mock_post
+
+    def _get_post_data(self, mock_post):
```

Comment:
Any reason you chose this request mocking approach over [HTTPretty](http://falcao.it/HTTPretty/)?